android/BOINC/app/src/main/java/edu/berkeley/boinc/ProjectsFragment.java

```diff
@@ -185,7 +185,7 @@ private void updateData(ArrayList<Project> latestRpcProjectsList, AcctMgrInfo ac
 			for(int x = 0; x < data.size(); x++) {
 				if(rpcResult.master_url.equals(data.get(x).id)) {
 					index = x;
-					continue;
+					break;
 				}
 			}
 			if(index == null) { // result is new, add
```

Comment:
I think all the comment/logging references to "result" or wrong here and likely a copy 'n' paste issue. Shouldn't all of those occurrences refer to "project" instead??? This doesn't really help understanding the original code's purpose.

Reply:
It looks fine to me.

Reply:
Seems fine to me.

Reply:
How? This code section doesn't deal with results at all but with projects. Just to be clear, this not about your change, it's an additional change that would be good to get in, since you're at it anyway.

Reply:
I agree with @brevilo. I believe comments and log messages should be changed from `result` to `project`

Reply:
Fixed.